_pytest/skipping.py

```diff
@@ -260,7 +266,7 @@ def pytest_report_teststatus(report):
     if hasattr(report, "wasxfail"):
         if report.skipped:
             return "xfailed", "x", "xfail"
-        elif report.failed:
+        elif report.passed:
```

Comment:
No need, that would be a normal failure: strict xfail now have report set to `failed`.